cypress/integration/drView_spec.js

```diff
@@ -0,0 +1,71 @@
+describe('Doctors Display', () => {
+
+  beforeEach(() => {
+    cy.fixture('mockDr.json')
+      .then(mockDr => {
+        cy.intercept('GET', 'https://head-to-toe-be.herokuapp.com/api/v1/medical_professionals?type=doctor&state=South Dakota', {
+          statusCode: 201,
+          delay: 100,
+          body: mockDr
+        })
+      })
+    cy.visit('http://localhost:3000/')
+  });
+
+  it('Should be able to open to the main page', () => {
+  cy.url().should('eq', 'http://localhost:3000/')
+  });
+
+  it('Should acknowledge the presence of list of of doctors with a sub title', () => {
+    cy.get('#doctors-button').should('be.visible')
+      .get('#doctors-button').click()
+      .location('pathname').should('eq', '/doctors')
+
+      .get('.dr-sub-title').should('be.visible')
+      .get('.dr-sub-title').should('contain', 'Doctor')
+      .get('.dr-sub-title').should('have.css', 'font-family', 'opendyslexic')
+  });
```

Comment:
This tests says it 'acknowledges the presence of a list', but the test seems to only test for the subtitle. We'll probably want to change the phrasing on this test

Reply:
Great idea